### ISSUE TITLE
Advanced Standing 

### ISSUE THREAD

**Issue by jkitchensSIUC** [CredentialEngine/vocabularies#432]
The military application of Advanced Standing includes:

- Training for a specific military occupation (has a code) (i.e. Military Credit Earned)
- Being at a level or grade within an occupation (e.g. E4, E5, E6, Ey, E8, E9)
- Experience (e.g. evaluation of educational and other experiences)
- Credit for specific course(s)
- By branch of the military

- Review the Military Bridge Program Roadmap, Branch: US Navy for School Name: Vincennes University http://www.learnmoreindiana.org/wp-content/uploads/2016/12/2016-11-30-HM-Pharmacy-Technician.pdf 
- http://www.learnmoreindiana.org/wp-content/uploads/2016/12/2016-11-30-68Q-Pharmacy-Technician.pdf
- See the A.S. in Pharmacy Technician https://credentialfinder.org/credential/1417/21st_Century_Skills_for_Workplace_Success 
- https://drive.google.com/file/d/0B1uSTaYpjhXJUEVpclhSM2IxRXM/view?usp=sharing

Advanced Standing description.
Description:
Military Bridge Program Roadmap:
Army:
Students who complete training for MOS/Rating: 68Q Pharmacy Specialist at the E4 level will receive 2 SH of credit for PFWL 100: Lifetime Fitness/Wellness and 2 SH of credit for Elective.
Students who complete training for MOS/Rating: 68Q Pharmacy Specialist at the E5 Level will receive 3 SH of credit for COMM 148: Interpersonal Communication, 2 SH of credit for PFWL 100: Lifetime Fitness/Wellness, and 2 SH of credit for Elective.
Students who complete training for MOS/Rating: 68Q Pharmacy Specialist at the E6-E9 level will receive 3 SH of credit for COMM 148: Interpersonal Communication, 2 SH of credit for PFWL 100: Lifetime Fitness/Wellness, 3 SH of credit for PHRM 200: Pharmacy Management, and 2 SH of credit for Elective.

Navy:
Students who complete MOS/Rating: HM Corpsman - Pharmacy Tech, Class C at the E4-E6 level will receive the following transfer credit: 3 SH for COMM 148: Interpersonal Communication, 3 SH for BIOL 108: Principles of Human A&P I, 2 SH for PFWL 100: Lifetime Fitness/Wellness, 3 SH for COMP 110: Intro to Computer Concepts, 3 SH for HIMT 110: Medical Terminology; 3 SH for PHRM 105: Pharmacology I, 3 SH for PHRM 106: Pharmacology II, 2 SH for PHRM 110: Dispensing Lab I, 3 SH for PHRM 120: Pharmacy Calculations, and 3 SH for PHRM 200: Pharmacy Management.
Students who complete MOS/Rating: HM Corpsman - Pharmacy Tech, Class C at the E7-E9 level will receive the following transfer credit: 3 SH for COMM 148: Interpersonal Communication, 3 SH for BIOL 108: Principles of Human A&P I, 3 SH for ENGL 108: Technical Writing, 2 SH for PFWL 100: Lifetime Fitness/Wellness, 3 SH for COMP 110: Intro to Computer Concepts, 3 SH for HIMT 110: Medical Terminology; 3 SH for PHRM 105: Pharmacology I, 3 SH for PHRM 106: Pharmacology II, 2 SH for PHRM 110: Dispensing Lab I, 3 SH for PHRM 120: Pharmacy Calculations, and 3 SH for PHRM 200: Pharmacy Management.

**Comment by siuc-nate** [comment]
@jkitchensSIUC and I discussed this today.  We have a few potential ideas in mind, such as:
- extend the properties of the existing condition profile to accommodate the data
- create a new class to accommodate the data
- consider handling of tabular data (via SKOS or similar mechanism)
- creating more properties/vocabularies (or extending existing ones) to accommodate military branches and ranks/levels (e.g., E4, E5, etc.)
- whether this also applies to other properties (e.g. recommends, preparation, etc)

**Comment by stuartasutton** [comment]
I think this is a straight forward mapping of one thing (military MOS) to course. Both should exist in machine actionable form (descriptions) that are then mapped.

I''ll fork up an example.

**Comment by siuc-nate** [comment]
Per our 9/26/2017 call:
- This is not directly a Registry issue
- This involves a different data model from Credentials, Competency Frameworks, etc.
- This could potentially be solved via an RDF Map object that enables third parties to make assertions
- This could potentially be solved via a W3C CSV encoding based model

**Comment by stuartasutton** [comment]
The W3C documents for CSVW:

>https://www.w3.org/TR/tabular-data-primer/
https://www.w3.org/ns/csvw

**Comment by siuc-nate** [comment]
Per @jkitchensSIUC, adding this as an example:
https://credentialengine.org/publisher/credential/1341/A_S__in_Emergency_Medical_Services
Here, Ivy Tech wants to include information about credit awarded for military training.

**Comment by stuartasutton** [comment]
As soon as we can schedule some **exclusive time**, we need to talk though some points from the examples at the top of this issue (restated at bottom of this message in pseudo data form). The mappings (whether 3rd party assertions or from a Credential like Ivy Tech's) involve subjects and objects that do not exist in the current CE context: (1) URI identified courses (which we chose not to include in credential descriptions); and (2) some entity embodying something such as "MOS/Rating: 68Q Pharmacy Specialist; E4" as a **surrogate** of a credential.  This question of whether these various bundles of military training and experience weren't, in fact, credentials (or should be treated as such) has been raised and should likely be faced. Then, with the exceptions of course URI, the mappings might well become relatively routine.

**Advanced Standing description**
# Military Bridge Program Roadmap
## ARMY:

MOS/Rating: 68Q Pharmacy Specialist at the E4 level
Receives:
2 SH of credit for PFWL 100: Lifetime Fitness/Wellness
2 SH of credit for Elective

MOS/Rating: 68Q Pharmacy Specialist at the E5 Level
Receives:
3 SH of credit for COMM 148: Interpersonal Communication
2 SH of credit for PFWL 100: Lifetime Fitness/Wellness
2 SH of credit for Elective

MOS/Rating: 68Q Pharmacy Specialist at the E6-E9 level
3 SH of credit for COMM 148: Interpersonal Communication
2 SH of credit for PFWL 100: Lifetime Fitness/Wellness
3 SH of credit for PHRM 200: Pharmacy Management
2 SH of credit for Elective

## NAVY:

MOS/Rating: HM Corpsman - Pharmacy Tech, Class C at the E4-E6 level 
Receives:
3 SH for COMM 148: Interpersonal Communication
3 SH for BIOL 108: Principles of Human A&P I
2 SH for PFWL 100: Lifetime Fitness/Wellness
3 SH for COMP 110: Intro to Computer Concepts
3 SH for HIMT 110: Medical Terminology
3 SH for PHRM 105: Pharmacology I
3 SH for PHRM 106: Pharmacology II
2 SH for PHRM 110: Dispensing Lab I
3 SH for PHRM 120: Pharmacy Calculations
3 SH for PHRM 200: Pharmacy Management

MOS/Rating: HM Corpsman - Pharmacy Tech, Class C at the E7-E9 level
Receives:
3 SH for COMM 148: Interpersonal Communication
3 SH for BIOL 108: Principles of Human A&P I
3 SH for ENGL 108: Technical Writing
2 SH for PFWL 100: Lifetime Fitness/Wellness
3 SH for COMP 110: Intro to Computer Concepts
3 SH for HIMT 110: Medical Terminology
3 SH for PHRM 105: Pharmacology I
3 SH for PHRM 106: Pharmacology II
2 SH for PHRM 110: Dispensing Lab I
3 SH for PHRM 120: Pharmacy Calculations
3 SH for PHRM 200: Pharmacy Management

**Comment by siuc-nate** [comment]
If we think about it like we'd think about any other case where a credential and a job were tightly coupled (i.e. being a nurse and having the license to be a nurse), I think it would make sense to create a credential for each relevant military occupation - not to consider the occupations themselves to _be_ credentials (or surrogates) any more than we consider a nursing job to be a credential.  The two are directly related, but distinct. The credential is the credential and the MOS is just an occupation code, just as there are codes for nursing.

Then everything else in the data model works as-is, I think.

In other words, you don't get the nursing license because you're a nurse; you get to be a nurse because you earned the license. Similarly, you don't get the credential for having the MOS, you get the MOS because you have the credential (the thing that proves you passed all the training to qualify for that MOS). The credential itself is the missing piece, here. Create it and everything else falls into place.

I could be missing something, but I don't think this requires a change to the data model at all.

**Comment by siuc-nate** [comment]
Per our 4-30-2019 meeting:
Closing this issue, as it has been conflated with other issues/solutions.
